scripts/common.Makefile

```diff
@@ -100,6 +100,9 @@ autoformat: ## runs black python formatter on this service's code. Use AFTER mak
 		--exclude "/(\.eggs|\.git|\.hg|\.mypy_cache|\.nox|\.tox|\.venv|\.svn|_build|buck-out|build|dist|migration|client-sdk|generated_code)/" \
 		$(CURDIR)
 
+.PHONY: mypy
+mypy: $(REPO_BASE_DIR)/mypy.ini ## runs mypy python static type checker on this services's code. Use AFTER make install-*
+	docker run -ti -v $(REPO_BASE_DIR)/mypy.ini:/config/mypy.ini -v $(CURDIR):/src --workdir=/src kiwicom/mypy mypy --config-file /config/mypy.ini src
```

Comment:
👍

Reply:
will do. I was probably dreaming of a makefile-only world... ;)